howdoi/howdoi.py

```diff
@@ -537,9 +538,9 @@ def _stash_remove(cmd_key, title):
     commands = keep_utils.read_commands()
     if commands is not None and cmd_key in commands:
         keep_utils.remove_command(cmd_key)
-        print(f'\n{BOLD}{GREEN}"{title}" removed from stash.{END_FORMAT}\n')
+        print(f'\n{BOLD}{GREEN} {title} removed from stash.{END_FORMAT}\n')
```

Comment:
Any reason to remove the `"` here? I think it's useful to show the name of query that was removed (or not found) in the stash.